### PR TITLE
added rating and textutility

### PR DESCRIPTION
**Pull request by vishalg8454**
Added 2 more components rating and textutility.

### CODE REVIEW

File: components/main.css [hunk 1]
```diff
@@ -5,3 +5,5 @@
 @import url('./avatar.css');
 @import url('./image.css');
 @import url('./input.css');
+@import url('./rating.css');
+@import url('./textutility.css');
```
Comment (kushanksriraj) [inline components/main.css]: Lets keep css file names hyphenated, `text-utility.css`?

File: docs.html [hunk 2]
```diff
@@ -283,12 +285,71 @@ <h2 class="component-title">Input</h2>
...
...seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522input-container%2522%253E%250A%2520%2520%2520%2520%253Clabel%2520class%253D%2522input-label%2522%2520for%253D%2522email-box%2522%253EEmail%253A%253C%252Flabel%253E%250A%2520%2520%2520%2520%253Cinput%2520class%253D%2522input-box%2522%2520type%253D%2522email%2522%2520name%253D%2522email-box%2522%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520placeholder%253D%2522enter%2520valid%2520email%2522%2520size%253D%252225%2522%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522input-container%2522%253E%250A%2520%2520%2520%2520%253Clabel%2520class%253D%2522input-label%2522%2520for%253D%2522msg-box%2522%253EMessage%253A%253C%252Flabel%253E%250A%2520%2520%2520%2520%253Ctextarea%2520class%253D%2522input-box%2522%2520type%253D%2522text%2522%2520name%253D%2522msg-box%2522%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520placeholder%253D%2522%28100%2520to%2520125%2520chars%29%2522%2520size%253D%252225%2522rows%253D%25223%2522%253E%250A%2520%2520%2520%2520%253C%252Ftextarea%253E%250A%253C%252Fdiv%253E"
```
Comment (kushanksriraj) [inline docs.html:288]: You can get a shorter url if you login to carbon.now.sh

File: components/textutility.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+}
+.text-md{
+    font-size: 1.25rem;
+    line-height: 1.5;
+    padding: 0.5rem;
+}
+.text-sm{
+    font-size: 1rem;
+    line-height: 1.5;
+    padding: 0.5rem;
```
Comment (kushanksriraj) [inline components/textutility.css]: padding value is same for many classes. Can we club them together?

File: docs.html [hunk 4]
```diff
@@ -283,12 +285,71 @@ <h2 class="component-title">Input</h2>
                         rows="3"></textarea>
                 </div>
                 <iframe class="component-code"
```
Comment (kushanksriraj) [inline docs.html:287]: Try using `embed` html tag.

File: docs.html [hunk 5]
```diff
@@ -283,12 +285,71 @@ <h2 class="component-title">Input</h2>
...
                     sandbox="allow-scripts allow-same-origin">
                 </iframe>
             </section>
+
+            <!-- rating -->
```
Comment (kushanksriraj) [inline docs.html:293]: Lets remove redundant comments, the code below states exactly what this section is.

File: docs.html [hunk 6]
```diff
@@ -283,12 +285,71 @@ <h2 class="component-title">Input</h2>
...
+                </div>
+                <iframe class="component-code"
+                    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522rating-container%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522rating-icon%2520checked%2520fas%2520fa-star%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522rating-icon%2520checked%2520fas%2520fa-star%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522rating-icon%2520checked%2520fas%2520fa-star%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522rating-icon%2520checked%2520fas%2520fa-star%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522rating-icon%2520fas%2520fa-star%2522%253E%253C%252Fi%253E%250A%253C%252Fdiv%253E"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </section>
+
+            <!-- textutility -->
+            <section id="textutility">
```
Comment (kushanksriraj) [inline docs.html:314]: `text-utility` ?
Also, no need for the comment above.

**Review by kushanksriraj — CHANGES_REQUESTED** [review]

**Review by Developers-codz — APPROVED** [review]
Looking good to me. Just do the suggested changes. Also there is typo in image section fix that
Screenshot (57)

File: components/main.css (hunk 1 above)
Comment (vishalg8454) [reply components/main.css]: done, edited.

File: docs.html (hunk 2 above)
Comment (vishalg8454) [reply docs.html:288]: I am using going via the "iframe" way and it seems to be giving long URL's only. Any way of shortening them?

File: components/textutility.css (hunk 3 above)
Comment (vishalg8454) [reply components/textutility.css]: I see only two values being repeated in three classes. Extracting them into a separate class would add another class so I let it be duplicated for now. Any suggestions?

File: docs.html (hunk 4 above)
Comment (vishalg8454) [reply docs.html:287]: Got to know this new HTML. I will research more into this and let iframe be the solution for now.

File: docs.html (hunk 6 above)
Comment (vishalg8454) [reply docs.html:314]: Yeah, will consider this.

**Comment by vishalg8454** [comment]
> Looking good to me. Just do the suggested changes. Also there is typo in image section fix that Screenshot (57)

Th

File: components/textutility.css (hunk 3 above)
Comment (kushanksriraj) [reply components/textutility.css]: Club the class names in css:

.class1 .class2 {
padding: 1rem;
}

File: docs.html (hunk 2 above)
Comment (kushanksriraj) [reply docs.html:288]: It's fine.

File: docs.html (hunk 4 above)
Comment (kushanksriraj) [reply docs.html:287]: Sure

**Pull request merged**